CHANGES.md

```diff
@@ -0,0 +1,28 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+Each change begins with a version (if applicable) and a timestamp, and then a set of changes, categorized by type:
+- Added for new features.
+- Changed for changes in existing functionality.
+- Deprecated for soon-to-be removed features.
+- Removed for now removed features.
+- Fixed for any bug fixes.
+- Security in case of vulnerabilities.
+
+
+## 2022-12-01
+
+### Changed
+
+- Added a template to CHANGES.md
+
+
+## 2022-02-28
```

Comment:
Should we start from 2022-12-01? Otherwise, it looks like nothing happened between February and December